graphql2/graphqlapp/schedule.go

```diff
@@ -81,14 +82,37 @@ func (q *Query) Schedule(ctx context.Context, id string) (*schedule.Schedule, er
 	return (*App)(q).FindOneSchedule(ctx, id)
 }
 
-func (s *Schedule) Shifts(ctx context.Context, raw *schedule.Schedule, start, end time.Time) ([]oncall.Shift, error) {
+func (s *Schedule) Shifts(ctx context.Context, raw *schedule.Schedule, start, end time.Time, userIDs []string) ([]oncall.Shift, error) {
 	if end.Before(start) {
 		return nil, validation.NewFieldError("EndTime", "must be after StartTime")
 	}
 	if end.After(start.AddDate(0, 0, 50)) {
 		return nil, validation.NewFieldError("EndTime", "cannot be more than 50 days past StartTime")
 	}
-	return s.OnCallStore.HistoryBySchedule(ctx, raw.ID, start, end)
+
+	err := validate.ManyUUID("userID", userIDs, 50)
+	if err != nil {
+		return nil, err
+	}
+
+	shifts, err := s.OnCallStore.HistoryBySchedule(ctx, raw.ID, start, end)
```

Comment:
Small comment, but should we avoid "magic numbers" like the 50 used here and above in `end.After(start.AddDate(0, 0, 50`?

Reply:
Looks like it's an issue with most of our calls to `validate.ManyUUID`, so I'm going to document that for its own issue, if possible